cpp/include/Ice/SSL/ServerAuthenticationOptions.h

```diff
@@ -77,7 +77,7 @@ namespace Ice::SSL
         /// cipher suite.
         /// @param info The connection info object that provides additional connection-related data. The
         /// `ConnectionInfo` type is an alias for the platform-specific connection info class.
-        /// @return true if the certificate chain is valid and the connection should proceed; false if the certificate
+        /// @return `true` if the certificate chain is valid and the connection should proceed; false if the certificate
```

Comment:
Also more in here too. You should try the regex I posted in my previous comment.
```suggestion
        /// @return `true` if the certificate chain is valid and the connection should proceed; `false` if the certificate
```